pkg/util/cluster/cluster.go

```diff
@@ -146,11 +146,6 @@ func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName str
 		visibility = api.VisibilityPrivate
 	}
 
-	fipsValidatedModules := api.FipsValidatedModulesEnabled
```

Comment:
Don't we want this to be able to continue to test FIPS in E2E?

Reply:
It still is tested. This change does remove passing in the parameter of FIPS being passed in through the preview API because it does not exist in the ClusterProfile for the preview API anymore. I instead set the FeatureFlag on the Subscription doc a couple of hundred of lines down: https://github.com/Azure/ARO-RP/pull/1768/files#diff-c7dba7bf3908043db4c458af9edd806998129f1ef7f58ee8fa54a312ce93f23cR453-R456

This does the trick, and I confirmed the  FIPS e2e test is running: https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=47348973&view=logs&j=10f66467-59e5-53bc-0c8a-42cc89770de4&t=3bbe7e73-d181-5aca-4b77-91acc0ef0f4a&l=1073